gen3_util/repo/cli.py

```diff
@@ -134,6 +136,8 @@ def init_cli(config, project_id: str):
             for _ in init(config, project_id):
                 logs.append(_)
 
+            click.secho(f"cloning {project_id}...", fg='green')
```

Comment:
change this to initializing @matthewpeterkort

Reply:
addressed in https://github.com/ACED-IDP/gen3_util/pull/42/commits/8b101bed29dbad86dfd16f83a2fb6f0a5523c85d